README.md

```diff
@@ -14,11 +14,11 @@ This repository has 3 main groups of files:
 
 ## Airflow Expectations
 
-These the Airflow expectations for these Funnel Cake DAGs to successfully run:
+The following are the Airflow expectations for the DAGs to successfully run:
 
 **Libraries & Packages**
 
-- Python 3.7
+- Python 3.9.10
```

Comment:
@cdoyle-temple Instead of stating a specific version, let's tell people to use the version defined in the .python-version file.